File: training-data/javascript/is_anagram.js
function isAnagram(a, b) {
  const normalize = (s) => s.replace(/\s/g, '').toLowerCase().split('').sort().join('');
  return normalize(a) === normalize(b);
}

console.log(isAnagram('listen', 'silent'));
console.log(isAnagram('hello', 'world'));

module.exports = { isAnagram };

File: training-data/go/is_anagram.go
package main

import (
	"fmt"
	"sort"
	"strings"
)

func isAnagram(a, b string) bool {
	normalize := func(s string) string {
		s = strings.ToLower(strings.ReplaceAll(s, " ", ""))
		chars := strings.Split(s, "")
		sort.Strings(chars)
		return strings.Join(chars, "")
	}
	return normalize(a) == normalize(b)
}

func main() {
	fmt.Println(isAnagram("listen", "silent"))
	fmt.Println(isAnagram("hello", "world"))
}
